ground/src/main/java/com/google/android/ground/model/geometry/Geometry.kt

```diff
@@ -110,6 +111,7 @@ data class LineString(val coordinates: List<Coordinates>) : Geometry {
  */
 @Serializable
 @SerialName("linear_ring")
+// TODO(#2499): Deprecate usages of LinearRing
```

Comment:
LinearRing is used by Polygon, incl. polygon proto. Sorry if my comment on chat was incorrect on this one.

Reply:
Done